### PR TITLE
Expand tilegrid to entire grid.

### PR DESCRIPTION
**Pull request by litghost**
With this PR, tilegrid now solves 100% for the following tiles on a7 and z7 settings:

- CLBL[LM]_[LR]
- HCLK_[LR]
- INT_[LR]
- BRAM_[LR]
- DSP_[LR]

On k7, most INT tiles are solved, expect for column X24 and X25 (#566).

### CODE REVIEW

**Comment by JohnDMcMaster** [comment]
are you aware of https://github.com/SymbiFlow/prjxray/issues/342

**Comment by litghost** [comment]
Yes, I'm already thinking about converting BRAM over to the fuzzer approach.

**Comment by JohnDMcMaster** [comment]
1) I would suggest merging that before this to avoid breaking BRAM 2) IMHO team needs to have a higher level discussion about current goals. This PR hurts what I consider to be MVP

File: fuzzers/005-tilegrid/generate_full.py [hunk 1]
```diff
@@ -468,6 +472,9 @@ def process_BLOCK_RAM(wordbase):
 
                     # FIXME: PCIE block cuts out some BRAM
                     # this messes up algorithm  as is and may cause this to fail
+                    if grid_y < 0:
```
Comment (JohnDMcMaster) [inline fuzzers/005-tilegrid/generate_full.py]: IIRC this isn't the right fix. It will generate a few tiles with bad addresses

**Comment by litghost** [comment]
I've successfully moved CLB and INT tiles for CLB, BRAM and IOB over to the fuzzer approach.  I need to add DSP, HCLK and DSP INT fuzzing to match what is currently in prxray-db.  The fuzzer approach is working nicely for things other than INT tiles.  INT tiles require some fairly routing specific logic, but it does demostrate that INT tiles appear to use the same bits regardless of whether it is attached to an CLB, IOB or BRAM.

**Comment by litghost** [comment]
@acomodi - The output of this PR populates all INT tiles I believe for zynq7.  In particular, it adds BRAM and DSP INT fuzzers which appears to add enough known INT tiles to solve the whole grid.  Kintex7 needs more work, as two columns remain unsolved.

**Comment by litghost** [comment]
Fixes #558.

**Comment by litghost** [comment]
Fixes #469.

**Comment by litghost** [comment]
Fixes #219.

**Comment by litghost** [comment]
This new tilegrid works for most fuzzers, but 100-dsp-makpat stopped working.  Investigating.

**Comment by litghost** [comment]
> I also see that the ps7_int fuzzer is actually not needed anymore because the INT_L base addresses are calculated from the adjacent INT_R. Therefore, I think the ps7_int can now be removed

I like the fact that there are more fuzzers than strictly speaking required.  It validates the the inputs are correct.   For example, I found a bug during development when the propagation found conflicting offsets in two instances.  Both cases were bugs in the fuzzers, rather than irregular INT columns.

**Comment by litghost** [comment]
Found and fixed issue where "height" was being used instead of "words".

**Pull request merged**